lib/invoker/power/dns.rb

```diff
@@ -31,7 +31,7 @@ def resource_class_matches?(resource_class)
       end
 
       def name_matches?(name)
-        name =~ /.*\.dev/
+        name =~ /.*\.#{Invoker::Power.tld_value}/
```

Comment:
So we really have 3 places from which we can fetch tld value. Why not just use `Invoker.config.tld` ?